app/GraphQL/Ecosystem/Queries/Roles/RoleAbilitiesQuery.php

```diff
@@ -21,4 +24,22 @@ public function getAllAbilities(mixed $root, array $query): array
 
         return $mapAbilities->all();
     }
+
+    public function getAllAbilitiesByRoles(mixed $root, array $request)
+    {
+        $roles = Bouncer::role()->where('name', $request['role'])->first();
```

Comment:
@FredPeal make sure to filter this by app

Reply:
Bouncer apply the filter on background